contrib/start_development_backend

```diff
@@ -4,6 +4,54 @@
 PERL5LIB=/vagrant/src/backend
 export PERL5LIB
 
+#prepare default configuration
```

Comment:
AFAIK: The file gets generated on initial configuration with the user 'Admin'. It is not generated during the package installation.